lib/web.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:io';
+import 'package:flutter_launcher_icons/abstract_platform.dart';
+import 'package:flutter_launcher_icons/icon_template.dart';
+import 'package:flutter_launcher_icons/utils.dart';
+import 'package:image/image.dart';
+import 'package:flutter_launcher_icons/constants.dart' as constants;
+
+final IconTemplateGenerator iconGenerator =
+    IconTemplateGenerator(defaultLocation: constants.webIconLocation);
+
+List<IconTemplate> webIcons = <IconTemplate>[
+  iconGenerator.get(name: 'Icon-192.png', size: 192),
+  iconGenerator.get(name: 'Icon-512.png', size: 512),
```

Comment:
Latest web template also contains `Icon-maskable-192.png` and `Icon-maskable-512.png` assets. Can we adjust this file to generate these two as well?